packages/editor/src/components/rich-text/index.js

```diff
@@ -136,7 +136,17 @@ export class RichText extends Component {
 	}
 
 	setRef( node ) {
-		this.editableRef = node;
+		if ( node ) {
+			const computedStyle = getComputedStyle( node );
```

Comment:
An option could be to only run this in dev environments and potentially throw an error, similar to the way the tooltip component does it:
https://github.com/WordPress/gutenberg/blob/f43572c8e55e692f45062ffaab0bdf8c8040b41b/packages/components/src/tooltip/index.js#L94-L97

That would avoid the need to run these extra checks in production.

Reply:
Updated, thanks!